src/main/com/example012/v1/model/StringConstant.java

```diff
@@ -0,0 +1,7 @@
+package main.com.example012.v1.model;
+
+public class StringConstant {
```

Comment:
Super. 
Tu víme, že budeme classu volat jen StringConstant.TABULATOR ... a nebudeme používat zbytečně:
 StringConstant stringConstants = new StringConstant()
zkus to udělat, jestli to bude možné ... pokud ano, přidej tam private constructor, aby to možné nebylo.

Reply:
Vytvořený private constructor

Reply:
ten empty line mit nemusel :D